appdaemon/DOCS.md

```diff
@@ -30,6 +30,8 @@ this isn't needed.
 
 ## Configuration
 
+Configurations are under `/addon_configs/a0d7b954_appdaemon`
```

Comment:
This doesn't have to be the case. The add-on simply requests a configuration folder (forks, and different stability channels will generate different folders).

That said, this is not an a specific thing for this add-on, but generic Home Assistnat logic and functionality.

../Frenck